core/domain/exp_domain.py

```diff
@@ -1396,6 +1393,13 @@ def update_interaction_fallbacks(self, fallbacks_list):
         self.interaction.fallbacks = [
             Fallback.from_dict(fallback_dict)
             for fallback_dict in fallbacks_list]
+        if self.interaction.fallbacks:
+            hint_list = []
+            for fallback in self.interaction.fallbacks:
+                if fallback.outcome.feedback:
+                    hint_list.append(
+                        Hint(fallback.outcome.feedback[0]).to_dict())
```

Comment:
Why the first one? (In other words, please comment why you choose the first one).